src/Apps/Collect/Routes/Collect/index.tsx

```diff
@@ -84,76 +85,80 @@ export const CollectApp: React.FC<React.PropsWithChildren<CollectAppProps>> = ({
         </Box>
 
         <Box>
-          <LazyArtworkGrid>
-            <SystemQueryRenderer<CollectArtworkFilterQuery>
-              query={graphql`
-                query CollectArtworkFilterQuery(
-                  $input: FilterArtworksInput
-                  $aggregations: [ArtworkAggregation]
-                  $shouldFetchCounts: Boolean!
-                ) {
-                  viewer {
-                    ...ArtworkFilter_viewer @arguments(input: $input)
-                    artworksConnection(
-                      aggregations: $aggregations
-                      input: $input
-                    ) {
-                      counts @include(if: $shouldFetchCounts) {
-                        followedArtists
-                      }
-                      aggregations {
-                        slice
-                        counts {
-                          value
-                          name
-                          count
+          <ArtworkGridContextProvider>
```

Comment:
I'm not sure if there is a difference wrapping the context provider here vs. one level up from `<ArtworkFilter>` below in the render function, but, intuitively, staying closer to the consumer of it seems more direct.